dev/traffic_router/run.sh

```diff
@@ -21,7 +21,6 @@ set -o errexit
 cd "$TC/traffic_router"
 
 mvn -Dmaven.test.skip=true compile -P \!rpm-build
-mvn -Dmaven.test.skip=true package -P \!rpm-build
```

Comment:
Looks like `mvn package` is required to generate `traffic_router/core/target/ROOT`. But the 2 commands can be combined to save 5 seconds like
```shell
mvn compile package
```